d2l-tspan.html

```diff
@@ -81,6 +124,16 @@
 				if (this.root.host) {
 					this.root.host.style.height = this.$.float.offsetHeight + 'px';
 				}
+			},
+			focusFeedback: function() {
+				if (this.focused && !this._feedbackInFocus) {
+					this._feedbackInFocus = true;
+				}
+			},
+			_focusOutHandler: function() {
+				if (this.focused) {
+					this._feedbackInFocus = false;
```

Comment:
Do we still need this? With the a `focused` attribute being the thing that defines how to render the component, I think we can get rid of the Javascript changes (aside from adding the `focused` property) and just have the CSS now, right?

Reply:
Actually, wait a minute, I think I had an idea that makes things much easier:

If we add a `focusin` and `focusout` event handler to this (which bubble up the the parent component), then it should be able to detect when a child element gets focus, then we can set the `focused` property accordingly, which will cause the CSS to work.

This way, the page using the tspan doesn't need to do anything- the webcomponent will take care of the focus all by itself